Missing importmode with pytest 6.0.0

`pytest` 6.0.0 just came out. `astropy` already sees this error:
```
_ ERROR collecting .tox/py38-test-alldeps/lib/python3.8/site-packages/astropy/conftest.py _
../../.tox/py38-test-alldeps/lib/python3.8/site-packages/pytest_doctestplus/plugin.py:172: in collect
    module = self.config.pluginmanager._importconftest(self.fspath)
E   TypeError: _importconftest() missing 1 required positional argument: 'importmode'
_ ERROR collecting .tox/py38-test-alldeps/lib/python3.8/site-packages/astropy/coordinates/tests/conftest.py _
../../.tox/py38-test-alldeps/lib/python3.8/site-packages/pytest_doctestplus/plugin.py:172: in collect
    module = self.config.pluginmanager._importconftest(self.fspath)
E   TypeError: _importconftest() missing 1 required positional argument: 'importmode'
_ ERROR collecting .tox/py38-test-alldeps/lib/python3.8/site-packages/astropy/table/tests/conftest.py _
../../.tox/py38-test-alldeps/lib/python3.8/site-packages/pytest_doctestplus/plugin.py:172: in collect
    module = self.config.pluginmanager._importconftest(self.fspath)
E   TypeError: _importconftest() missing 1 required positional argument: 'importmode'
_ ERROR collecting .tox/py38-test-alldeps/lib/python3.8/site-packages/astropy/wcs/tests/conftest.py _
../../.tox/py38-test-alldeps/lib/python3.8/site-packages/pytest_doctestplus/plugin.py:172: in collect
    module = self.config.pluginmanager._importconftest(self.fspath)
E   TypeError: _importconftest() missing 1 required positional argument: 'importmode'
_ ERROR collecting .tox/py38-test-alldeps/lib/python3.8/site-packages/astropy/wcs/wcsapi/conftest.py _
../../.tox/py38-test-alldeps/lib/python3.8/site-packages/pytest_doctestplus/plugin.py:172: in collect
    module = self.config.pluginmanager._importconftest(self.fspath)
E   TypeError: _importconftest() missing 1 required positional argument: 'importmode'
______________________ ERROR collecting docs/conftest.py _______________________
../../.tox/py38-test-alldeps/lib/python3.8/site-packages/pytest_doctestplus/plugin.py:172: in collect
    module = self.config.pluginmanager._importconftest(self.fspath)
E   TypeError: _importconftest() missing 1 required positional argument: 'importmode'
```

Example log: https://travis-ci.org/github/astropy/astropy/jobs/712696735

--- a/.travis.yml
+++ b/.travis.yml
@@ -44,7 +44,7 @@ jobs:
 install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda.sh
-    - python ./setup.py install
+    - python -m pip install .
 
 script:
    - $PYTEST_COMMAND --doctest-plus

--- a/pytest_doctestplus/plugin.py
+++ b/pytest_doctestplus/plugin.py
@@ -10,6 +10,8 @@ import re
 import sys
 import warnings
 
+from packaging.version import Version
+
 import pytest
 
 from pytest_doctestplus.utils import ModuleChecker
@@ -20,6 +22,8 @@ try:
 except ImportError:  # PY2
     def indent(text, prefix):
         return '\n'.join([prefix + line for line in text.splitlines()])
+
+PYTEST_GT_5 = Version(pytest.__version__) > Version('5.9.9')
 
 comment_characters = {
     '.txt': '#',
@@ -169,7 +173,12 @@ def pytest_configure(config):
             if self.fspath.basename == "setup.py":
                 return
             elif self.fspath.basename == "conftest.py":
-                module = self.config.pluginmanager._importconftest(self.fspath)
+                if PYTEST_GT_5:
+                    module = self.config.pluginmanager._importconftest(
+                        self.fspath, self.config.getoption("importmode"))
+                else:
+                    module = self.config.pluginmanager._importconftest(
+                        self.fspath)
             else:
                 try:
                     module = self.fspath.pyimport()

--- a/pytest_doctestplus/utils.py
+++ b/pytest_doctestplus/utils.py
@@ -5,7 +5,7 @@ import operator
 import re
 import subprocess
 import sys
-from distutils.version import LooseVersion
+from packaging.version import Version
 
 import pkg_resources
 
@@ -24,8 +24,6 @@ class ModuleChecker:
         ).decode()
         packages = {item['name'].lower(): item['version']
                     for item in json.loads(packages)}
-        import pprint
-        pprint.pprint(packages)
         return packages
 
     def compare_versions(self, v1, v2, op):
@@ -39,7 +37,7 @@ class ModuleChecker:
         if op not in op_map:
             return False
         op = op_map[op]
-        return op(LooseVersion(v1), LooseVersion(v2))
+        return op(Version(v1), Version(v2))
 
     def _check_distribution(self, module):
         """

--- a/pytest_doctestplus/utils.py
+++ b/pytest_doctestplus/utils.py
@@ -19,6 +19,8 @@ class ModuleChecker:
 
     def get_packages(self):
         packages = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze']).decode().splitlines()
+        import pprint
+        pprint.pprint(packages)
         packages = [package.split('==') for package in packages if '==' in package]
         return {name.lower(): version for name, version in packages}
 

--- a/pytest_doctestplus/utils.py
+++ b/pytest_doctestplus/utils.py
@@ -1,4 +1,5 @@
 import importlib.util
+import json
 import logging
 import operator
 import re
@@ -18,11 +19,14 @@ class ModuleChecker:
         self.packages = {}
 
     def get_packages(self):
-        packages = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze']).decode().splitlines()
+        packages = subprocess.check_output(
+            [sys.executable, '-m', 'pip', 'list', '--format', 'json']
+        ).decode()
+        packages = {item['name'].lower(): item['version']
+                    for item in json.loads(packages)}
         import pprint
         pprint.pprint(packages)
-        packages = [package.split('==') for package in packages if '==' in package]
-        return {name.lower(): version for name, version in packages}
+        return packages
 
     def compare_versions(self, v1, v2, op):
         op_map = {

--- a/pytest_doctestplus/utils.py
+++ b/pytest_doctestplus/utils.py
@@ -1,3 +1,4 @@
+import importlib.util
 import logging
 import operator
 import re
@@ -5,22 +6,16 @@ import subprocess
 import sys
 from distutils.version import LooseVersion
 
+import pkg_resources
+
 logger = logging.getLogger(__name__)
 
 
 class ModuleChecker:
     def __init__(self):
-        if LooseVersion(sys.version) < LooseVersion('3.4'):
-            import imp
-            self._find_module = imp.find_module
-            self._find_distribution = self._check_distribution
-            self.packages = self.get_packages()
-        else:
-            import importlib.util
-            import pkg_resources
-            self._find_module = importlib.util.find_spec
-            self._find_distribution = pkg_resources.require
-            self.packages = {}
+        self._find_module = importlib.util.find_spec
+        self._find_distribution = pkg_resources.require
+        self.packages = {}
 
     def get_packages(self):
         packages = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze']).decode().splitlines()

